applications/openshift/api-server/api_server_authorization_mode/rule.yml

```diff
@@ -2,31 +2,32 @@ documentation_complete: true
 
 prodtype: ocp3
 
-title: 'Disable use of AlwaysAllow for the API Server Authorization Mode'
+title: 'Ensure API Server authorization-mode is set to Webhook'
 
 description: |-
-    The OpenShift API server should not allow anonymous API requests.
+    By default, unauthenticated/unauthorized users have no access to OpenShift nodes
+    and the API Server <tt>authorization-mode</tt> is set to <tt>Webhook</tt>.
     To ensure that the API server requires authorization for API requests,
-    validate that <tt>AlwaysAllow</tt> is not configured for <tt>authorization-mode</tt>
+    validate that <tt>authorization-mode</tt> is is configured to <tt>Webhook</tt>
     in <tt>/etc/origin/master/master-config.yaml</tt>:
     <pre>kubernetesMasterConfig:
       apiServerArguments:
         authorization-mode:
-        - AlwaysAllow</pre>
+        - Webhook</pre>
 
 rationale: |-
-    The <tt>AlwaysAllow</tt> authorization mode allows all requests including
-    anonymous requests. This would grant any authenticated entity full cluster
-    access.
+    If <tt>authorization-mode</tt> is configured ensure the value is set to <tt>Webhook</tt> The
+    node systemd service (<tt>atomic-openshift-node</tt>) will not start if the configuration
+    value is set to anything other than <tt>Webhook</tt>.
```

Comment:
@shawndwells I would maybe change this to: 
```If <tt>authorization-mode</tt> is not configured to <tt>Webhook</tt>, the node systemd service (<tt>atomic-openshift-node</tt>) will not start.```

Of course, it would be also nice to have a security rationale to it.....

Reply:
Updated!